features/convergedcloud/exec.config

```diff
@@ -3,12 +3,10 @@ set -Eeuo pipefail
 
 systemctl enable ignition-disable.service
```

Comment:
`systemctl enable ignition-disable.service` should happen in the feature it installs if it is intended as default behaviour.

Reply:
Let's focus on the cri-tools here and move this in another PR. I created #17 for this.